### PR TITLE
MESH-861 - More configuration updates

### PR DESCRIPTION
**Pull request by adamdossa**
- for operators, makes the controller account a master key, and stash account a signing key of the same identity
- use a consolidated config function (v1_testnet_genesis) for all v1 configs
- clean up some configs and small fixes
- remove CDD claim from register_did

### CODE REVIEW

File: scripts/cli/log.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+./node_modules/.bin/pm2 logs all
```
Comment (maxsam4) [inline scripts/cli/log.sh:1]:
```suggestion
npx pm2 logs all
```

File: scripts/cli/run.sh [hunk 2]
```diff
@@ -22,3 +22,4 @@ cleanup
 sleep 2
 
 ./node_modules/.bin/pm2 start environment.config.js --only "pmesh-peer-node-1,pmesh-peer-node-2"
```
Comment (maxsam4) [inline scripts/cli/run.sh:24]:
```suggestion
npx pm2 start environment.config.js --only "pmesh-peer-node-1,pmesh-peer-node-2"
```

File: scripts/cli/stop.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+./node_modules/.bin/pm2 stop all
```
Comment (maxsam4) [inline scripts/cli/stop.sh:1]:
```suggestion
npx pm2 stop all
```

File: src/chain_spec.rs [hunk 4]
```diff
@@ -97,6 +102,10 @@ pub fn get_authority_keys_from_seed(
     (
         get_account_id_from_seed::<sr25519::Public>(&format!("{}//stash", seed)),
         get_account_id_from_seed::<sr25519::Public>(seed),
+        // get_from_seed::<GrandpaId>(&format!("{}//gran", seed)),
+        // get_from_seed::<BabeId>(&format!("{}//babe", seed)),
+        // get_from_seed::<ImOnlineId>(&format!("{}//imon", seed)),
+        // get_from_seed::<AuthorityDiscoveryId>(&format!("{}//auth", seed)),
```
Comment (maxsam4) [inline src/chain_spec.rs:108]: Clean up extra comments?

Comment (adamdossa) [reply src/chain_spec.rs:108]: I'm going to leave these in for now. One slightly annoying thing is that when `get_authority_keys_from_seed` is called, for e.g. Alice, it uses the same seed for all of the session keys. This makes switching it for the "real" keys a bit of a pain as it isn't easy to differentiate between them. Adding an additional derivation path works for this, but then local nodes based on Alice etc. don't work as the node doesn't include these additional keys.

There are ways to fix this, but not a top priority, so leaving this in, in the short term.

**Comment by adamdossa** [comment]
> Need to change
> 
> ```
> impl IsV1Network for ChainSpec {
>     fn is_v1_network(&self) -> bool {
>         self.name().starts_with("Polymesh V1")
>     }
> }
> ```
> 
> to
> 
> ```
> impl IsV1Network for ChainSpec {
>     fn is_v1_network(&self) -> bool {
>         self.name().starts_with("Polymesh Aldebaran")
>     }
> }
> ```
> 
> Without this right runtime will never trigger.

Updated

**Pull request merged**